tests/settings/test_settings.py

```diff
@@ -18,3 +18,28 @@ def test_invalid_storage() -> None:
         assert False, f"'validate_settings' raised an exception {exc}"
     finally:
         cluster[0]["storage_sets"].remove("non_existing_storage")
+
+
+def test_topics_sync() -> None:
```

Comment:
You should probably explain what exactly this test is testing because I don't have a clear picture just glancing at the code

Reply:
or maybe it could be titled something like `test_all_topics_in_settings_validator()`?

Reply:
Good point, I'll add some comments and rename test.